tasks.py

```diff
@@ -3,12 +3,14 @@
 import os
 
 
+# serve a local development server
```

Comment:
I have a lizard-brain dislike of comments that just say what the function/variable/etc already says
```suggestion
```

Reply:
Me too, my bad. I've actually never worked with Poetry before so I left this in when I was internally noting what everything does. Oopsies >_<